doc/api/tls.md

```diff
@@ -796,17 +796,17 @@ and their processing can be delayed due to packet loss or reordering. However,
 smaller fragments add extra TLS framing bytes and CPU overhead, which may
 decrease overall server throughput.
 
-## tls.checkServerIdentity(host, cert)
+## tls.checkServerIdentity(hostname, cert)
```

Comment:
The corresponding change needs to also be made in `lib/tls.js` so that argument names in error messages and stack traces match those in documentation.

Reply:
@Trott Made changes to `lib/tls.js` as well